docs/manual/called-methods-checker.tex

```diff
@@ -6,8 +6,12 @@
 of the form ``call method A before method B''. For the purpose of this
 checker, a method has ``definitely been called'' if it is \emph{invoked}:
 a method that might never return or that might throw an exception has
-a method that might never return or that might throw an exception has
 definitely been called on every path after the call, including exceptional paths.
+The checker also ignores the possibility that a run-time exception might occur:
```

Comment:
As of right now, the assumptions do not extend to all built-in run-time exceptions.  E.g., `ClassCastException` and `ArrayIndexOutOfBoundsException` are not completely ignored.  Are you thinking we will ignore such exceptions in a future change?  If not, maybe we should be specific here about which exact exception types are ignored.

Reply:
https://github.com/typetools/checker-framework/pull/4869 will add `Error` and `RuntimeException`, which was what I was thinking about when I wrote this. Maybe this text shouldn't mention that, though, and instead only focus on nullness.

Reply:
I rewrote the text again to only mention nullness. I think this version is a bit better, but I'm still not that happy with it.